tests/db_util.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+from sqlalchemy import text
+import sqlalchemy
+
+
+def query_pg(engine: sqlalchemy.engine.Engine, query_str: str) -> list[dict[str, Any]]:
+    with engine.connect() as connection:
+        result = connection.execute(text(query_str))
+        rows = result.fetchall()
+
+    return [
+        {
+            # key: (bytes(value) if isinstance(value, memoryview) else value)
+            key: (f"0x{value.hex()}" if isinstance(value, memoryview) else value)
+            for key, value in zip(result.keys(), row)
+        }
+        for row in rows
+    ]
+
+
+# def query_pg(engine: sqlalchemy.engine.Engine, query_str: str) -> list[dict[str, Any]]:
```

Comment:
Commented out code :)

Reply:
I thought I had already pushed that out. But maybe it's still local.

Reply:
Ya this is long gone. You gotta be looking at something else.

Reply:
I looked at the diff view here in GH and I suppose it was outdated. Sorry, my bad.